spectral_cube/base_class.py

```diff
@@ -205,6 +205,19 @@ def world(self, view):
 
         return world[::-1]  # reverse WCS -> numpy order
 
+    def flattened_world(self, view=()):
+        """
+        Retrieve the world coordinates corresponding to the extracted flattened
+        version of the cube
+        """
+
+        # NOTE: this should be moved to SpatialCoordMixinClass once masks
+        # are implemented for lower dim objects - EK
```

Comment:
This is in `SpatialCoordMixinClass` now, no?

Reply:
Yup! will remove.